kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/utils/TestUtils.scala

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.delta.kernel.defaults.utils
+
+import java.util.{Optional, TimeZone}
+
+import scala.collection.JavaConverters._
+import scala.collection.mutable.ArrayBuffer
+
+import io.delta.kernel.{Scan, Snapshot, Table}
+import io.delta.kernel.client.TableClient
+import io.delta.kernel.data.Row
+import io.delta.kernel.defaults.client.DefaultTableClient
+import io.delta.kernel.types._
+import io.delta.kernel.utils.CloseableIterator
+import org.apache.hadoop.conf.Configuration
+import org.scalatest.exceptions.TestFailedException
+import org.scalatest.Assertions
+
+trait TestUtils extends Assertions {
+
+  lazy val defaultTableClient = DefaultTableClient.create(new Configuration())
```

Comment:
make the `new Configuraiton` a method, so that individual suite can override it if needed.